<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Atlas</class>
 <widget class="QMainWindow" name="Atlas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>853</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="iniciarButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>70</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>INICIAR</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>541</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="graphLayout"/>
   </widget>
   <widget class="QPushButton" name="pausarButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>200</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>PAUSAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pararButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>330</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>PARAR</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>853</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
